pkg/env/env.go

```diff
@@ -0,0 +1,123 @@
+// Package for processing environment variables.
+package env
+
+// TODO: we need to add tests for this package.
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// DefaultEnvVariables set $PATH, $TERM and $container.
+var DefaultEnvVariables = map[string]string{
+	"PATH":      "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+	"TERM":      "xterm",
+	"container": "podman",
+}
+
+const whiteSpaces = " \t"
```

Comment:
Should there be a space in front of the tab?  I'm not sure.

Reply:
Yes, this is either a space or a tab.

Reply:
OK, I'm not sure I was clear, should this be `"\t"` or `"{tab or space}\t"` ?

Reply:
It's used in `strings.TrimLeft` and meant to trim _tabs and whitespaces_, so we're good. `TrimLeft` interprets the string as a set of characters (in contrast to a string/sequence) for trimming.